pkgs/development/python-modules/Wand/default.nix

```diff
@@ -7,12 +7,12 @@
 , psutil
 , memory_profiler
 , pytest_xdist
-, sharedLibraryExtension
 }:
 
 let
-  magick_wand_library = "${imagemagick}/lib/libMagickWand-6.Q16${sharedLibraryExtension}";
-  imagemagick_library = "${imagemagick}/lib/libMagickCore-6.Q16${sharedLibraryExtension}";
+  soext = stdenv.hostPlatform.extensions.sharedLibrary;
+  magick_wand_library = "${imagemagick}/lib/libMagickWand-6.Q16${soext}";
```

Comment:
These aren't equivalent; `sharedLibraryExtension` had a dot in it, whereas `soext` doesn't.

Reply:
soext does have a dot.

Reply:
Oh sorry!

Reply:
No worries! It does look odd because it didn't have the dot before the anti-quote before, like most.